src/nvim/highlight_group.c

```diff
@@ -177,6 +177,8 @@ static const char *highlight_init_both[] = {
   "default link PmenuKindSel     PmenuSel",
   "default link PmenuSbar        Pmenu",
   "default link ComplMatchIns    NONE",
+  "default link ComplHint        NonText",
+  "default link ComplSuffix      Title",
```

Comment:
Is there a reason 'colors/vim.lua' and this have different link for `ComplSuffix`? The `MoreMsg` seems like a better highlight group (despite it being designed for command line).